cookiecutter.json

```diff
@@ -13,5 +13,7 @@
     "use_celery": "n",
     "use_maildump": "n",
     "use_sentry": "n",
+    "use_docker": "n",
```

Comment:
I'm -1 on introducing on introducing the `use_docker` question here. People will end up with all the docker specific files anyway because there is no way to exclude files.

They will have the files but no further documentation for it.

The current implementation integrates nicely in the eco system we already have. A user can switch between all available options (virtualenv or docker) without the need to change anything. For the time being, we should make sure that we keep these side by side without introducing too much switches.